src/ui/screens/AgencyQR.tsx

```diff
@@ -38,8 +51,11 @@ export default class AgencyQR extends React.Component<QRProps, QRState> {
             connectionError: "",
             verifying: false,
             isConnectionReady: false,
+            agent_connected: window.localStorage.getItem('agent_connected') || 'false',
+            connectionId: window.localStorage.getItem('connection_id') || ''
```

Comment:
Using `getItem() || ...` could be abstracted as getItem("...', defaultValue) as a service to library consumers.